code/game/objects/items/control_wand.dm

```diff
@@ -28,7 +28,7 @@
 			mode = WAND_EMERGENCY
 		if(WAND_EMERGENCY)
 			mode = WAND_OPEN
-	to_chat(user, "Now in mode: [mode].")
+	balloon_alert(user, "Mode: [mode]")
```

Comment:
I dont like how this is the only one using : compared to all the other ones which are Mode set to [mode] or something.

Reply:
eh
```suggestion
	balloon_alert(user, "Mode set to [mode]")
```